reverse-proxy/traefik-statuspage.yml

```
@@ -6,4 +6,4 @@ services:
     image: traefik
```

```suggestion
    image: traefik:v1.7-alpine
```

I started it this way:

```
OWNCLOUD_DOMAIN=mytest.shniq.cloud \
OWNCLOUD_ADMIN_USERNAME=admin \
OWNCLOUD_ADMIN_PASSWORD=very-very-secret \
docker-compose \
    -f owncloud-base.yml \
    -f owncloud-official.yml \
    -f owncloud-exported-ports.yml \
    -f cache/redis.yml \
    -f database/mariadb.yml \
    -f reverse-proxy/traefik.yml \
    -f reverse-proxy/traefik-statuspage.yml \
    up -d
```

Without the 1.7 tag, I get this error:

```
traefik_1   | 2020/01/29 22:11:52 command traefik error: field not found, node: redirect
compose-playground_traefik_1 exited with code 1
```

---

Done